migrations/1663008177719_rename-jobRole-to-role.js

```diff
@@ -0,0 +1,9 @@
+/* eslint-disable camelcase */
```

Comment:
we might not need this for the migrations so far. do let me know if we do

Reply:
Deleted @jocrah

Reply:
cool, looks good, i think the two other files still have it. not really looked through much if we do need it but kindly check that as well. thanks

Reply:
@jocrah sorted for all

Reply:
okay @itzgeebee,then its behaving quite strange, are we yet to push because still shows for [this](https://github.com/tbf-challenge/teamwork-be/blob/ee7546f2039332d91b692a640bf6ed0066ae4b93/migrations/1663008177719_rename-jobRole-to-role.js)  and [this](https://github.com/tbf-challenge/teamwork-be/blob/ee7546f2039332d91b692a640bf6ed0066ae4b93/migrations/1663008659109_add-default-role-as-user.js)  in the file changes

Reply:
I really don't know why it's behaving that way. I tried adding and committing again and it's saying everything is up to date @jocrah

Reply:
that's cool @itzgeebee, not really a blocker so we can move forward with it